src/openrct2/interface/InteractiveConsole.cpp

```diff
@@ -938,6 +942,18 @@ static int32_t cc_set(InteractiveConsole& console, const utf8** argv, int32_t ar
             }
             console.Execute("get cheat_disable_support_limits");
         }
+        else if (strcmp(argv[0], "current_rotation") == 0 && invalidArguments(&invalidArgs, int_valid[0]))
+        {
+            uint8_t currentRotation = get_current_rotation();
+            rct_window* mainWindow = window_get_main();
+            // there are only 4 possible rotations (0-3)
+            int32_t newRotation = int_val[0] & 3;
```

Comment:
Perhaps it would be better to reject arguments out of range and report usage in such case. It wouldn't let users develop bad habits of passing invalid values.